sdk/appservice/arm-appservice/review/arm-appservice.api.md

```diff
@@ -3468,11 +3468,12 @@ export interface GitHubActionWebAppStackSettings {
 }
 
 // @public
-export interface Global {
+interface Global_2 {
     getDeletedWebApp(deletedSiteId: string, options?: GlobalGetDeletedWebAppOptionalParams): Promise<GlobalGetDeletedWebAppResponse>;
     getDeletedWebAppSnapshots(deletedSiteId: string, options?: GlobalGetDeletedWebAppSnapshotsOptionalParams): Promise<GlobalGetDeletedWebAppSnapshotsResponse>;
     getSubscriptionOperationWithAsyncResponse(location: string, operationId: string, options?: GlobalGetSubscriptionOperationWithAsyncResponseOptionalParams): Promise<void>;
 }
+export { Global_2 as Global }
```

Comment:
Not sure what this is about?

Reply:
I remember seeing similar, strange behavior in latest version of api-extractor before?